packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -149,10 +154,21 @@ export function newProject(options: OptionsSchema): Rule {
 					}
```

Comment:
If the below call to the `upgradeIgniteUIPackages` method is set here, I believe you won't need to call `setVirtual` again